### PR TITLE
Cisco ESA

### PR DESCRIPTION
**Pull request by jonathantauman**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [x] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CiscoESAIronPortEmailAPI/Integrations/CiscoIronPortEMailAPI/CiscoIronPortEMailAPI.py
Comment (dorschw) [inline Packs/CiscoESAIronPortEmailAPI/Integrations/CiscoIronPortEMailAPI/CiscoIronPortEMailAPI.py:9]: Please replace with
```python
import urllib3
urllib3.disable_warnings()
```

File: Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2_test.py
Comment (dorschw) [inline Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2_test.py:84]: please add type hints to all arguments in the file

**Review by idovandijk — CHANGES_REQUESTED** [review]
Thank you @jonathantauman for the content contribution!
Please refer to my comment on the other PR you made: https://github.com/demisto/content/pull/21338/#pullrequestreview-1152297725

File: Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2.py
Comment (dorschw) [inline Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2.py:1]: Please remove this line

**Comment by ShirleyDenkberg** [comment]
Please give me edit/write permissions for the files in this PR so that I can review the files properly.

File: Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2.yml
Comment (ShirleyDenkberg) [inline Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2.yml:489]: why is this recipient/sender?  The argument is for sender_list.  Shouldn't it just be sender?  And is it just the addresses that are deleted or the sender in general?

File: Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2.yml
Comment (ShirleyDenkberg) [inline Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2.yml:485]: why is this recipient/sender?  The argument is for recipient_list.  Shouldn't it just be recipient?  And is it just the addresses that are deleted or the recipient in general?

**Comment by ShirleyDenkberg** [comment]
@jonathantauman Please regenerate the CiscoEmailSecurityApplianceIronPortV2 Readme since I made several changes in the yml file.  BTW, I noticed that the following 2 integration parameters appear in the CiscoEmailSecurityApplianceIronPortV2.yml file but not in the README.
- IncidentType
- isFetch
(And just want to give you a thumbs up on the pack readme.  Really nice.)

@dorschw @idovandijk @bakatzir @GuyLibermanPA Doc review completed.

File: Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2.yml
Comment (jonathantauman) [reply Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2.yml:489]: This argument implementation is according to the API usage, and was discussed with Meital.

File: Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2.yml
Comment (jonathantauman) [reply Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2.yml:485]: This argument implementation is according to the API usage, and was discussed with Meital.

**Comment by idovandijk** [comment]
@jonathantauman are you finished with the changes?

File: Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2.py
Comment (dorschw) [inline Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2.py:72]: please use the `ok_codes` argument in `_http_request` to make sure the request returns a valid response (rather than 401, 403 etc), here as well as elsewhere in the integration. This way, invalid responses will raise an error (please make sure the error message is shown properly)

**Review by idovandijk — DISMISSED** [review]
As in the SMA PR, please update the release notes and provide a screenshot of the layout in an incident.
After that I will happily approve so we can go forward and merge :)

**Comment by idovandijk** [comment]
@dorschw done my part, feel free to approve when you're done.

**Review by dorschw — APPROVED** [review]

**Pull request merged**
